index.html

```diff
@@ -111,32 +111,82 @@
+    <dd>The user agent is using a Bluetooth connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>cellular</dfn></dt>
+    <dd>The user agent is using a cellular connection as the <a>underlying connection technology</a> (e.g., EDGE, HSPA, LTE, etc.).</dd>
+    <dt><dfn>ethernet</dfn></dt>
+    <dd>The user agent is using an Ethernet connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>none</dfn></dt>
+    <dd>The user agent will not contact the network when the user follows links or when a script requests a remote page (or knows that such an attempt would fail) - i.e., equivalent to `navigator.onLine === false` in HTML.</dd>
+    <dt><dfn>wifi</dfn></dt>
+    <dd>The user agent is using a Wi-Fi connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>wimax</dfn></dt>
+    <dd>The user agent is using a WiMAX connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>other</dfn></dt>
+    <dd>The user agent is using a connection type that is not one of enumerated connection types as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>mixed</dfn></dt>
+    <dd>The user agent is using multiple connection types as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn>unknown</dfn></dt>
+    <dd>The user agent has established a network connection, but is unable to determine what is the <a>underlying connection technology</a>.</dd>
+  </dl>
+  </section>
+
+  <section data-dfn-for="EffectiveConnectionType" data-link-for="EffectiveConnectionType">
+  ### Effective connection types
+  This section defines the <dfn data-lt="effective connection type">effective connection types</dfn> (ECT):
+
+  <table id="effective-connection-type-table" >
+    <caption><dfn>Table of effective connection types</dfn></caption>
+    <thead>
+      <tr>
+        <th>ECT</th>
+        <th>Minimum RTT (ms)</th>
+        <th>Maximum downlink (Kbps)</th>
+        <th>Explanation</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><dfn>slow2g</dfn></td>
+        <td>1900</td>
+        <td>50</td>
+        <td>The network is suited for small transfers only such as text-only pages.</td>
+      </tr>
+      <tr>
+        <td><dfn>2g</dfn></td>
+        <td>1300</td>
+        <td>70</td>
+        <td>The network is suited for transfers of small images.</td>
+      </tr>
+      <tr>
+        <td><dfn>3g</dfn></td>
+        <td>200</td>
+        <td>700</td>
+        <td>The network is suited for transfers of large assets such as high resolution images, audio, and SD video.</td>
+      </tr>
+      <tr>
+        <td><dfn>4g</dfn></td>
+        <td>0</td>
+        <td>∞</td>
+        <td>The network is suited for HD video, real-time video, etc.</td>
+      </tr>
+    </tbody>
+  </table>
+  <p class="note">The above thresholds are generated from real user observations by Chrome over different connection types and network technologies (e.g., EDGE, GPRS).</p>
```

Comment:
Is there any public document that describes the observations made by the chrome team? That is, something we can cite?